### PR TITLE
add some funi but real this time.

### PR DESCRIPTION
**Pull request by vadd7**
gay

### CODE REVIEW

File: PinguUnfryer.py [hunk 1]
```diff
@@ -128,34 +136,157 @@ def UnfryQ():
...
+
+                print("step 2 completed successfully...")
+
+                print("processing final step")
+
+                for i in range(0, 20):
+                    print(f"progress: {i * 10}%")
+                    time.sleep(randint(10, 40) / 10)
+
+                key = input(pyfiglet.figlet_format("OH FUCK I ALMOST FORGOT ABOUT THIS. WHATS THE SECRET KEY?"))
```
Comment (dawutheuwad) [inline PinguUnfryer.py:218]: no swears pwese

File: PinguUnfryer.py [hunk 2]
```diff
@@ -128,34 +136,157 @@ def UnfryQ():
...
+
+                print("ok sorry my bad for taking this long")
+                print("here is your answer bib big mann't")
+
+                time.sleep(1)
+                print(decoded_text)
+                exit()
+
+            case "cook bred":
+                print("we have bread idiot.")
```
Comment (dawutheuwad) [inline PinguUnfryer.py:234]:
```suggestion
                print("we have bread guy.")
```
optional

File: PinguUnfryer.py [hunk 3]
```diff
@@ -103,6 +106,11 @@ def ProgramKill2():
     time.sleep(3)
     exit()
 
+def MisteryKill():
+    print("JAJAJAJA YOU DUMB IDIOT YOU DONT KNOW THE MYSTERY MAN CODE HEHEHEHEHEHAHAHEHAHAEH GO EAT A HEDGEHOG!+!+!+!343")
```
Comment (dawutheuwad) [inline PinguUnfryer.py:110]: footage of me eating hedgehog (real):
🦔😋

**Review by dawutheuwad — COMMENTED** [review]
need some editing, but will be merged

File: PinguUnfryer.py (hunk 3 above)
Comment (vadd7) [reply PinguUnfryer.py:110]: very yummy you should try

**Pull request closed without merging**